### PR TITLE
29 aug

### PR DESCRIPTION
**Pull request by Kapil0187**

### CODE REVIEW

File: app/sidekiq/my_first_job.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class MyFirstJob
+  include Sidekiq::Job
+
+  def perform()
+
```
Comment (rishabhharod) [inline app/sidekiq/my_first_job.rb:5]: what's the functionality of this method

File: config/environments/development.rb [hunk 2]
```diff
@@ -84,16 +84,17 @@
...
+  
   config.action_mailer.smtp_settings = {
   address:              'smtp.gmail.com',
   port:                 587,
   domain:               'example.com',
-  user_name:            ENV['EMAIL_USERNAME'],  # Use environment variable
-  password:             ENV['EMAIL_PASSWORD'],  # Use App Password from Gmail
+  user_name:            'kbarsker@bestpeers.com',  # Use environment variable
+  password:             'rdvv zhrq zzsu lzvq',  # Use App Password from Gmail
   authentication:       'plain',
```
Comment (rishabhharod) [inline config/environments/development.rb:94]: is password contain white spaces?

File: config/routes.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
   get 'trains/index'
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  root to: "home#rootpage" 
+  root to: "home#root_page" 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
```
Comment (rishabhharod) [inline config/routes.rb:7]: be care full about naming convention whenever you define files name and methods name

File: config/environments/development.rb (hunk 2 above)
Comment (Kapil0187) [reply config/environments/development.rb:94]: password is not contain white spaces, I am using gmail app password pas key for the security.

File: app/sidekiq/my_first_job.rb (hunk 1 above)
Comment (Kapil0187) [reply app/sidekiq/my_first_job.rb:5]: I forget to remove this method. now i will remove.

File: config/routes.rb (hunk 3 above)
Comment (Kapil0187) [reply config/routes.rb:7]: okay i will make sure. thanks for review.

**Pull request merged**